docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -303,29 +303,27 @@ You can only edit the settings of Data Assets created in GX Cloud.
 
 3. Click **Save**.
 
-## Secure your GX API Data Source connection strings
+## Data Source credential management
 
-When you use the GX API and not GX Cloud to connect to Data Sources, you can obfuscate sensitive Data Source credentials in your connection string as an additional security measure.
+To connect to your database in GX Cloud, there are two methods for managing credentials:
 
-1. Store your credential value as an environment variable by entering `export ENV_VAR_NAME=env_var_value` in the terminal or adding the command to your `~/.bashrc` or `~/.zshrc` file. For example:
+1. **Direct input**: You can input credentials directly into GX Cloud. These credentials are securely encrypted at rest and in transit, stored in our database, and accessible to the GX Agent and any API user.
```

Comment:
Credentials are accessible by any API user? This reads that credentials are not securely stored. (And is this true?)

I'd suggest:

These credentials are stored in GX Cloud and securely encrypted at rest and in transit. When Data Source credentials have been directly provided, they can be used to connect to a Data Source in any GX Cloud deployment pattern.

Reply:
I believe it is true. from Lukasz: "inputting credentials directly into GX Cloud is the simplest way to get started. These would be stored in our database, encrypted at rest and in transit. They would be available to the GX Agent, as well as any API user."

but I agree what you said sounds better, thanks - made the update. I guess the API user would still need to have credentials to use